lib/kredis/types/proxy.rb

```diff
@@ -1,27 +1,31 @@
 class Kredis::Types::Proxy
+  require "active_support/core_ext/module"
```

Comment:
I'd move the require to `lib/kredis.rb`:

https://github.com/rails/kredis/blob/2469a8d6f166ec6b4331f339eb128a3dcf315945/lib/kredis.rb#L2